sapp/ui/filters.py

```diff
@@ -171,6 +172,32 @@ def export_filter(
             raise
 
 
+class ServeExportFilter(View):
```

Comment:
The rest of the app runs over graphql. Why did we need to switch to a view here? It's not a blocker to landing this PR, per se, but I want to be sure we're not adding a new paradigm without a good reason. Each new way of querying the server just increases the maintenance overhead a bit.

Reply:
Okay, so sorry for finding a new approach. We can do it this PR three ways:
 - use the filter on the frontend (this seems the best approach as we already query the saved filter before export) (this was migrated to use this approach)
 - use filter along with backend as shown in this PR
 
Making an additional backend query on click of `a` to download file doesn't seem to work as often the file gets downloaded before the results arrive from the backend :/ But however, there is an option to display an additional pop-up saying the filter is ready, but then that kinda makes the UX bad. I was out of my Witts here and went with the additional view which imo is also kinda bad as you mentioned.

Reply:
I am fine with this approach. The view is more suited to deliver files since it easily give us a URL we can embed in a <a> tag to download the file (GET /export_filter/whatever). As @abishekvashok  points out using GraphQL the solution would be more involved requiring onclick event submitting a POST request to graphQL and then locally downloading the file. Compared to the solution 1. this will guarantee us that the export functionality is common between the cli and the UI and doesn't rely on the clientside implementation of the currentFilter React JS props :)